### PR TITLE
Added CI and proper implementation

### PR DESCRIPTION
**Pull request by subho007**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/appknox/appknox.github.io/pull/17" target="_blank">ReviewNinja

File: Gemfile [hunk 1]
```diff
@@ -2,3 +2,5 @@ source 'https://rubygems.org'
 gem 'jekyll'
 gem 'github-pages'
 gem 'reduce'
+gem "html-proofer"
```
Comment (subho007) [inline Gemfile]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

**Comment by subho007** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[](https://sideci.com/github_repositories/55060189/auto_correct/rubocop/pull_request/17/branch/ci)

See the documentation if you want to get more information about auto-correct.
[Document](https://sideci.com/ja/docs/autofix-rubocop)

**Comment by subho007** [comment]
Created Auto-correct Pull Request from [SideCI](https://sideci.com/)

[https://github.com/appknox/appknox.github.io/pull/18](https://github.com/appknox/appknox.github.io/pull/18)

File: Rakefile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 require "rubygems"
-require "Reduce"
+require "reduce"
```
Comment (subho007) [inline Rakefile]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

**Comment by subho007** [comment]
Created Auto-correct Pull Request from [SideCI](https://sideci.com/)

[https://github.com/appknox/appknox.github.io/pull/19](https://github.com/appknox/appknox.github.io/pull/19)

File: Rakefile [hunk 3]
```diff
@@ -1,45 +1,45 @@
...
+      File.open(file, 'w') do |f|
+        f.write(min)
+      end
+      compressed += File.size(file)
+    else
+      puts "Skipping: #{file}"
       end
   end
-  puts "Total compression %0.2f\%" % (((original-compressed)/original)*100)
+  puts 'Total compression %0.2f%' % (((original - compressed) / original) * 100)
```
Comment (subho007) [inline Rakefile:46]: **[rubocop]**
- Favor `format` over `String#%`. [:ref](https://github.com/bbatsov/ruby-style-guide#sprintf)

File: Rakefile [hunk 4]
```diff
@@ -1,45 +1,47 @@
...
```
Comment (subho007) [inline Rakefile:46]: **[rubocop]**
- Favor `format` over `String#%`. [:ref](https://github.com/bbatsov/ruby-style-guide#sprintf)

**Comment by subho007** [comment]
Auto-correctable file didn't exist on this Pull Request from [SideCI](https://www.sideci.com/).

**Comment by subho007** [comment]
Auto-correctable file didn't exist on this Pull Request from [SideCI](https://www.sideci.com/).

**Comment by subho007** [comment]
Auto-correctable file didn't exist on this Pull Request from [SideCI](https://www.sideci.com/).

**Comment by subho007** [comment]
@appknox/frontend There are so many mistakes here

```
- ./_site/company/index.html
  *  ID  already defined (line 1613)
  *  ID  already defined (line 1652)
  *  Unexpected end tag : p (line 184)
- ./_site/find-out-now/index.html
  *  ID  already defined (line 518)
  *  ID  already defined (line 557)
  *  Opening and ending tag mismatch: section and div (line 241)
- ./_site/get-in-touch/index.html
  *  ID  already defined (line 579)
  *  ID  already defined (line 540)
- ./_site/index.html
  *  ID  already defined (line 797)
  *  ID  already defined (line 836)
  *  Unexpected end tag : p (line 189)
- ./_site/pricing/index.html
  *  ID  already defined (line 738)
  *  ID  already defined (line 699)
  *  Unexpected end tag : div (line 338)
  *  Unexpected end tag : form (line 333)
  *  Unexpected end tag : li (line 249)
  *  Unexpected end tag : p (line 180)
  *  Unexpected end tag : p (line 177)
  *  Unexpected end tag : ul (line 334)
- ./_site/resources/index.html
  *  ID  already defined (line 582)
  *  ID  already defined (line 543)
  *  ID res-in-all already defined (line 187)
  *  ID res-in-case-study already defined (line 189)
  *  ID res-in-ebook already defined (line 195)
  *  ID res-in-infographic already defined (line 193)
  *  ID res-in-report already defined (line 191)
  *  Unexpected end tag : p (line 180)
- ./_site/secure-your-app/index.html
  *  ID  already defined (line 547)
  *  ID  already defined (line 586)
```

**Pull request merged**